packages/cashier/src/modules/cashier-onboarding/components/cashier-onboarding-side-notes/cashier-onboarding-side-note-fiat.tsx

```diff
@@ -16,11 +17,7 @@ const CashierOnboardingSideNoteFiat: React.FC = observer(() => {
                         is_from_derivgo ? (
                             <span />
                         ) : (
-                            <a
-                                key={0}
-                                className='link link--orange'
-                                onClick={() => window.LiveChatWidget?.call('maximize')}
-                            />
+                            <a key={0} className='link link--orange' onClick={() => Chat.open()} />
```

Comment:
```suggestion
                            <a key={0} className='link link--orange' onClick={Chat.open} />
```
just a suggestion, up to you @prince-deriv  . 
you can change everywhere if you think it is worth doing it :)